workshop/content/docs/publishing/ogcapi-records.md

```diff
@@ -69,40 +69,77 @@ develop your own custom pygeoapi plugin, or convert your metadata to OGC API - R
 
     If you do not have Python installed, consider running this exercise in a Docker container. See the [Setup Chapter](../setup.md#using-docker-for-python-clients).
 
+    === "Linux/Mac"
```

Comment:
I think the indentation needs to be:

### Sample ISO 19139 to TinyDBCatalogue loader

It is possible to load more example ISO19139 metadata in a TinyDB database with [the following script](https://github.com/geopython/pygeoapi/blob/master/tests/load_tinydb_records.py) ([raw](https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_tinydb_records.py)):

=== "Linux/Mac"

    <div class="termy">
    ```bash
    cd workshop/exercises/data/tartu/metadata
    curl -O https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_tinydb_records.py
    python3 load_tinydb_records.py xml catalogue.tinydb
    ```
    </div>

=== "Windows (PowerShell)"

    <div class="termy">
    ```bash
    cd workshop/exercises/data/tartu/metadata
    curl https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_tinydb_records.py
    python3 load_tinydb_records.py xml catalogue.tinydb
    ```
    </div>